packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -294,7 +294,13 @@ async function handler(
     })
   );
 
-  const eventTimeZone = eventType.schedule?.timeZone ?? null;
+  const user = eventType.users.find((user) => user.id === eventType.userId);
+
+  const userSchedule = user?.schedules.find(
+    (schedule) => !user?.defaultScheduleId || schedule.id === user?.defaultScheduleId
+  );
+
```

Comment:
Why you add this check `!user?.defaultScheduleId`? it means that  defaultScheduleId is falsy, this part of the condition evaluates to true. I don't think it's needed. The task is to set user's default timezone as fallback. pls remove it

Reply:
@anikdhabal It would mean that if user's default schedule ID is null (which in the first place is not possible), but still if it becomes possible, then it will just consider the first schedule of the user and it's timezone.

If we remove this condition, then suppose the default schedule ID of the user is null, then none of the schedule will be selected and we will not be able to retrieve the timezone.

Please let me know what do you think.

Reply:
Yeah, I understand why you added this, but our task here is to set defaultscheduleid as the fallback. There is very little chance that both the event type schedule timezone and the user’s default schedule ID is null at the same time. Also, we don't want to set any of the user’s schedule timezones other than default. So for now, please remove it; we will think about it later.

Reply:
Ok @anikdhabal ! Done! 👍